content/kitchen/_index.md

```diff
@@ -1,10 +1,28 @@
 +++
-title = "Test Kitchen"
-linkTitle = "Test Kitchen"
+title = "Installing Test Kitchen Enterprise"
+linkTitle = "Installing Test Kitchen Enterprise"
 
 [menu.kitchen]
-title = "Overview"
+title = "Installing Test Kitchen Enterprise"
 identifier = "kitchen/overview"
 parent = "kitchen"
 weight = 10
 +++
+
+To install the new Chef Test Kitchen Enterprise, first ensure you have the latest Habitat installer. For details on how to do this, refer to the following section: [Hab-based builds]({{< relref "hab_builds" >}}).
+
+Then use the Habitat installer to get the latest package from the Habitat Builder site:
+
+```sh
+hab pkg install chef/test-kitchen/3.7.1
```

Comment:
```suggestion
sudo hab pkg install --binlink --force chef/chef-test-kitchen-enterprise
```

Reply:
This needs needs to be updated since user will be installing new chef-test-kitchen-enterprise